### PR TITLE
Beta protocol cherry pick fixes

### PR DESCRIPTION
**Pull request by Siegrift**
These changes seem to fix the monorepo build.

I've also ran the unit tests and the protocol ones seem to pass, but node tests fail (but I think that is expected).
There are also some TS errors . I'll look all of those issues later (next week)

### CODE REVIEW

**Review by amarthadan — COMMENTED** [review]
Changes on parts I was working with looks good to me :+1:

**Comment by Siegrift** [comment]
@andreogle @acenolaza @bbenligiray Could you take a look, please?

I think it would be best to merge this and continue working on master.

The plan is to rebase-merge this into `beta-protocol-cherry-pick` and then rebase that branch onto `master` (because some new commits have been added in the meantime).

File: packages/node/src/handlers/test-api.ts [hunk 1]
```diff
@@ -34,17 +34,17 @@ export async function testApi(
 
   const workerOpts: WorkerOptions = {
     cloudProvider: config.nodeSettings.cloudProvider,
-    airnodeIdShort: wallet.getAirnodeIdShort(masterHDNode),
+    airnodeAddressShort: wallet.getAirnodeAddressShort(masterHDNode.address),
```
Comment (acenolaza) [inline packages/node/src/handlers/test-api.ts:37]: On line 16 it should be calling `wallet.getAirnodeWallet(config) `and here in line 37 we should be passing the address of the airnode wallet instead.

**Review by acenolaza — COMMENTED** [review]
A few questions and comments

File: packages/deployer/src/evm/index.ts [hunk 2]
```diff
@@ -30,8 +30,8 @@ export async function checkAirnodeParameters(config: Config, airnodeId: string,
         try {
```
[the remark this answers is not in the document]
Comment (Siegrift) [reply packages/deployer/src/evm/index.ts:30]: I included this commit on https://github.com/api3dao/airnode/pull/529 and rebased this branch on top of that.

File: packages/node/src/handlers/start-coordinator.ts [hunk 3]
```diff
@@ -55,9 +55,9 @@ async function coordinator(config: Config): Promise<CoordinatorState> {
   });
   logger.info('Forking to initialize providers complete', logOptions);
 
-  const hasNoRequests = state2.providerStates.evm.every((evmProvider) =>
-    hasNoActionableRequests(evmProvider!.requests)
-  );
+  const hasNoRequests = state2.providerStates.evm.every((evmProvider) => {
+    return hasNoActionableRequests(evmProvider!.requests);
+  });
```
[the remark this answers is not in the document]
Comment (Siegrift) [reply packages/node/src/handlers/start-coordinator.ts:60]: I think I was debugging something and forgot to remove.

File: packages/node/src/handlers/test-api.ts (hunk 1 above)
Comment (Siegrift) [reply packages/node/src/handlers/test-api.ts:37]: Why is the test passing actually? o.O

File: packages/node/src/evm/initialization.test.ts [hunk 4]
```diff
@@ -1,585 +0,0 @@
-import { mockEthers } from '../../test/mock-utils';
```
[the remark this answers is not in the document]
Comment (Siegrift) [reply packages/node/src/evm/initialization.test.ts:1]: removed.

File: packages/node/src/evm/requests/event-logs.ts [hunk 5]
```diff
@@ -44,8 +44,6 @@ export async function fetch(options: FetchOptions): Promise<EVMEventLog[]> {
     fromBlock,
     toBlock: options.currentBlock,
     address: options.address,
-    // Ethers types don't support null for a topic, even though it's valid
-    // @ts-ignore
```
[the remark this answers is not in the document]
Comment (Siegrift) [reply packages/node/src/evm/requests/event-logs.ts:48]: Yes, I think one of the updates did fix the typings.

File: packages/node/src/providers/initialize.test.ts [hunk 6]
```diff
@@ -1,147 +0,0 @@
-const spawnAwsMock = jest.fn();
```
[the remark this answers is not in the document]
Comment (Siegrift) [reply packages/node/src/providers/initialize.test.ts:1]: removed

**Review by andreogle — APPROVED** [review]

File: packages/deployer/src/evm/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (acenolaza) [reply packages/deployer/src/evm/index.ts:30]: shouldn't this file be gone?

File: packages/node/src/evm/handlers/initialize-provider.test.ts [hunk 7]
```diff
@@ -15,6 +15,7 @@ import * as fixtures from '../../../test/fixtures';
 
 describe('initializeProvider', () => {
   it('fetches, maps and authorizes requests', async () => {
+    jest.setTimeout(30_000);
```
Comment (acenolaza) [inline packages/node/src/evm/handlers/initialize-provider.test.ts:18]: do we want to leave this?

File: packages/node/src/handlers/start-coordinator.test.ts [hunk 8]
```diff
@@ -30,6 +30,7 @@ import * as fixtures from '../../test/fixtures';
 
 describe('startCoordinator', () => {
   it('fetches and processes requests', async () => {
+    jest.setTimeout(30000);
```
Comment (acenolaza) [inline packages/node/src/handlers/start-coordinator.test.ts:33]: do we want to leave this too?

**Review by acenolaza — APPROVED** [review]
Couldn't find anything major 🏆 
Just left a few questions and please squash while merging to have a clearer git log 🙏🏻

File: packages/node/src/evm/handlers/initialize-provider.test.ts (hunk 7 above)
Comment (Siegrift) [reply packages/node/src/evm/handlers/initialize-provider.test.ts:18]: Yes, I had some problems running this locally. It doesn't hurt to add this anyway.

**Comment by Siegrift** [comment]
> Just left a few questions and please squash while merging to have a clearer git log 🙏🏻

I forgot, but I'll squash the commits when merging the other branch to master.

**Pull request merged**